src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleCreationService.java

```diff
@@ -88,6 +115,105 @@ public void bulkCreationLogic(String userToken, MultipleDetails multipleDetails,
 
     }
 
+    private void addLegalRepsFromSinglesCases(MultipleDetails multipleDetails) throws IOException {
+        MultipleData multipleData = multipleDetails.getCaseData();
+        String leadCaseId = multipleData.getLeadCaseId();
+        List<String> caseIds = multipleData.getCaseIdCollection().stream()
+            .map(o -> o.getValue().getEthosCaseReference())
+            .collect(Collectors.toList());
+
+        if (!isNullOrEmpty(leadCaseId)) {
+            caseIds.add(leadCaseId);
```

Comment:
The lead case should already be part of the caseIdCollection, no?

Reply:
You're right, didn't spot this, thanks!